### DISCUSSION TITLE
Option to stop removing `dependencies` from `package.json`

### DISCUSSION THREAD

**Discussion by brettz9** [npm/feedback#227] [CLI]
Hi,

I like to have `dependencies` even if empty in package.json. I can search through a large file to find it, and it just makes clear the status.

However, `npm i` regularly removes this when empty. Could we get config to stop this from happening?

**Comment by ljharb** [comment]
I would prefer this as well; it minimizes diff churn when a dependency is added or removed. (same for every dep category)

**Comment by MylesBorins** [comment]
@brettz9 if you have an idea for a specific feature to be added to the npm CLI the best next step would be to open an issue over at https://github.com/npm/rfcs to propose the change to the CLI team!

Thanks for opening the discussion.
